angular-client/src/components/charging-switch/charging-switch.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'charging-switch',
+  templateUrl: './charging-switch.component.html',
+  styleUrls: ['./charging-switch.component.css']
+})
+export default class ChargingSwitch implements OnInit {
+  ngOnInit() {
+    // TODO: figure out how we want to hook up the charging switch (and what data to link it too)
```

Comment:
again, Jack question for the specific topic to use here, but based on what he sent, it'll just be a boolean that corresponds to this switch (it should only be displaying the data, you won't be able to click it or anything, tbh we may wanna move away for the switch, but we can decide that after all the data is hooked up). Jack's Comment:
Is balancing
Is charging enabled (could be a switch we operate)
Voltage of pack
Faults of charger box, BMS
BMS high cell value temp, low cell value, temp
BMS request (editable in future) /charger box draw/ bms measure current
BMS balance delta (high cell V minus low cell V)